apps/testing/src/main/java/com/akto/testing/TestExecutor.java

```diff
@@ -455,7 +456,9 @@ public List<TestingRunResult> startWithLatch(
                 if (!testingRunResults.isEmpty()) {
                     trim(testingRunResults);
                     TestingRunResultDao.instance.insertMany(testingRunResults);
+                    ApiInfoDao.instance.updateLastTestedField(testingRunResults, apiInfoKey);
```

Comment:
don't call this for all testingRunResults

Reply:
need clearance.